docs/usage/shoot/access_restrictions.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Access Restrictions
```

Comment:
I think this feature is only related to compliance, so can we move it under `docs/usage/compliance/access_restrictions.md`?

Reply:
I don't see a `compliance` folder yet. Also, I'm not convinced creating one now and moving the doc there. You could argue that the maintenance doc talking about automatic version upgrades is also related to compliance. However, it is in `usage/shoot`.
Generally, I don't fully understand the structure now. For example, I think it's very subjective whether something is considered "advanced" or not (why is `shoot_cleanup.md` in `advanced`, while `shoot_scheduling_profiles.md` is not)? Anyways, let's not discuss this topic here.